{
    "topic": "TS Dataset Type Standard Converter Error",
    "pageNumber": "4",
    "type": "TS Dataset Type Standard Converter Error - Get Context Variables Failed",
    "definition": {
        "text": "It was not possible to get the context variables needed for running this process. See below for details.",
        "updated": 1621859784437,
        "translations": [
            {
                "language": "TR",
                "text": "Özet: Bu işlemi çalıştırmak için gereken bağlam değişkenlerini elde etmek mümkün değil. Ayrıntılar için aşağıya bakın.",
                "updated": 1647587400460
            }
        ]
    },
    "paragraphs": [
        {
            "style": "Error",
            "text": "Get Context Variables Failed",
            "updated": 1621860374194,
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata: Bağlam Değişkenlerini Alma Başarısız",
                    "updated": 1647587408327
                }
            ]
        },
        {
            "style": "Title",
            "text": "Error Info",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Bilgileri",
                    "updated": 1647587417236
                }
            ]
        },
        {
            "style": "Text",
            "text": "In the following section, you will find the information recorded when the error occurred.",
            "translations": [
                {
                    "language": "TR",
                    "text": "Aşağıdaki bölümde, hata oluştuğunda kaydedilen bilgileri bulacaksınız.",
                    "updated": 1647587425326
                }
            ]
        },
        {
            "style": "Text",
            "text": "You will only see the information if you opened this page as a result of the error actually occurring.",
            "updated": 1611680880668,
            "translations": [
                {
                    "language": "TR",
                    "text": "Bilgileri yalnızca bu sayfayı gerçekten oluşan hatanın bir sonucu olarak açtıysanız göreceksiniz.",
                    "updated": 1647587433459
                }
            ]
        },
        {
            "style": "Subtitle",
            "text": "Error Message",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Mesajı",
                    "updated": 1647587440908
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Message of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Mesajı aşağıdadır:",
                    "updated": 1647587448656
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorMessage"
        },
        {
            "style": "Subtitle",
            "text": "Error Stack",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Yığını",
                    "updated": 1647587458349
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Stack of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Yığını aşağıdadır:",
                    "updated": 1647587466422
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorStack"
        },
        {
            "style": "Subtitle",
            "text": "Error Code",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Kodu",
                    "updated": 1647587475412
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the Error Code of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın Hata Kodu aşağıdadır:",
                    "updated": 1647587481536
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorCode"
        },
        {
            "style": "Subtitle",
            "text": "Error Details",
            "translations": [
                {
                    "language": "TR",
                    "text": "Hata Detayları",
                    "updated": 1647587493403
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following are all the Error details of the Exception that occurred at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) meydana gelen İstisnanın tüm Hata ayrıntıları aşağıdadır:",
                    "updated": 1647587500570
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "errorDetails"
        },
        {
            "style": "Subtitle",
            "text": "Node Name",
            "translations": [
                {
                    "language": "TR",
                    "text": "Düğüm Adı",
                    "updated": 1647587519381
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the node's name of the Node most related to this error:",
            "translations": [
                {
                    "language": "TR",
                    "text": "Aşağıda, bu hatayla en çok ilişkili olan Düğümün ( Node ) adı verilmiştir:",
                    "updated": 1647587531258
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "nodeName"
        },
        {
            "style": "Subtitle",
            "text": "Node Code",
            "translations": [
                {
                    "language": "TR",
                    "text": "Düğüm Kodu",
                    "updated": 1647587537764
                }
            ]
        },
        {
            "style": "Text",
            "text": "The following is the code stored at the node that produced the error at the API Data Fetcher Bot:",
            "translations": [
                {
                    "language": "TR",
                    "text": "API Veri Alıcı Botunda ( API Data Fetcher Bot ) hatayı üreten düğümde depolanan kod aşağıdadır:",
                    "updated": 1647587545281
                }
            ]
        },
        {
            "style": "Placeholder",
            "text": "nodeCode"
        }
    ]
}